sigar_linux.go

```diff
@@ -388,3 +440,21 @@ func readProcFile(pid int, name string) ([]byte, error) {
 
 	return contents, err
 }
+
+/* For SCSI and IDE devices, only display devices and not individual partitions.
+   For other major numbers, show all devices regardless of minor (for LVM, for example).
+   As described here: http://www.linux-tutorial.info/modules.php?name=MContent&pageid=94 */
```

Comment:
Hopefully this is clearer? I know the negative in the name is -1, but `isDevice` didn't seem clear. I had to go back and iron it out, it turns out we need info from `/dev/partitions` to find out which devices are interesting to us.